convert/src/config.py

```diff
@@ -70,27 +78,49 @@ def get_request_session(*args: List[Any], **kwargs: Dict[str, Any]) -> Session:
 
 
 settings = Settings()
+logger_.info(f"{settings.s3_credentials_provider=}")
+
+
+class NotConfiguredException(Exception):
+    pass
+
+
+def create_boto3_config():
+    boto3_config = {}
+    if settings.s3_credentials_provider == "minio":
+        logger_.info(
+            f"S3_Credentials provider - {settings.s3_credentials_provider}"
+        )
+        boto3_config.update(
+            {
+                "aws_access_key_id": settings.minio_access_key,
+                "aws_secret_access_key": settings.minio_secret_key,
+                "endpoint_url": settings.minio_host,
+            }
+        )
+    elif settings.s3_credentials_provider == "aws_iam":
+        logger_.info(
+            f"S3_Credentials provider - {settings.s3_credentials_provider}"
+        )
```

Comment:
Let's move logging to a single line (L110)

Reply:
agreed